go.mod

```diff
@@ -1,39 +1,39 @@
 module github.com/kuadrant/kuadrant-operator
 
-go 1.22
+go 1.22.0
```

Comment:
this was intended?

Reply:
Yes, as of 1.22 the toolchain directive is added if the go version is not specific. I.e. if I run `go mod tidy` with go version 1.22.5 it updates the go.mod to the following:

```
go 1.22.0

toolchain go1.22.5
```
The only way to avoid is adding the full version afaict

Reply:
TIL this. Thanks.